partials/_floating.scss

```diff
@@ -0,0 +1,142 @@
+.floating {
+    margin-top: 52px;
+    width: 100%;
+    display: flexbox;
+    flex-direction: column;
+    position: relative;
+    margin: auto;
+    
+
+    @include media ( ">=md" ) {
+        text-align: center;
+        padding: 0;
+        max-width: 768px;
+    }
+
+    @include media ( ">xl" ) {
+        max-width: 1024px;
+    }
+
+    &__block {
+        flex-direction: row;
+        max-height: fit-content;
+        margin-top: 150px;
+        height: fit-content;
+        width: 100%;
+        text-align: right;
+        
+        @include media ( ">md" ) {
+            margin-top: 0px;
+        }
+    }   
+
+    &__title {
+        color: $tws_coral;
+        font-family: $font-handwritten;
+        transform: rotate(-90deg);
+        font-weight: 400;
+        line-height: 49px;
+        position: relative;
+        font-size: 37.3px;
```

Comment:
is the comma value really needed? We usually use integers for font-size.

Reply:
Have changed it to 37px, think I got 37.3px from what I saw on live...